be/src/vec/functions/least_greast.cpp

```diff
@@ -174,6 +174,12 @@ struct FunctionFieldImpl {
         auto& res_data = static_cast<ColumnInt32*>(result_column)->get_data();
 
         const auto& column_size = arguments.size();
+        for (int i = 1; i < column_size; ++i) {
+            block.replace_by_position(
+                    arguments[i],
+                    ColumnConst::create(block.get_by_position(arguments[i]).column, 1));
+        }
```

Comment:
no need convert it, could set use_default_implementation_for_constants() = true.

Reply:
For the `field` funcition, it uses `FunctionMultiSameArgs`, for which `use_default_implementation_for_constants ` already returns `true`.

The problems is, in function `PreparedFunctionImpl::default_implementation_for_constant_arguments`, it unpacks the const arguments of `field` function, so we need to pack it back to `ConstColumn` here to simplify the modification.

Reply:
set = false